lakefile.lean

```diff
@@ -62,11 +62,21 @@ package mathlib where
 ## Mathlib libraries
 -/
 
+target nolints_style (pkg) : System.FilePath :=
+  inputTextFile <| pkg.dir / "scripts" / "nolints-style.txt"
+
+target no_lints_prime_decls (pkg) : System.FilePath :=
+  inputTextFile <| pkg.dir / "scripts" / "no_lints_prime_decls.txt"
+
 @[default_target]
 lean_lib Mathlib where
   leanOptions := mathlibLeanOptions
   -- Mathlib also enforces these linter options, which are not active by default.
   moreServerOptions := mathlibOnlyLinters
+  extraDepTargets := #[
```

Comment:
Optional: add a comment why this is here? (Perhaps this is sufficiently obvious... I'm not sure. Up to you.)

Reply:
Done.